src/component/menu/Header/Header.tsx

```diff
@@ -103,6 +114,17 @@ const Header: VFC = () => {
         );
     }
 
+    const onSetMode = () => {
```

Comment:
Could move this into `UIContext.setMode` or a custom hook.

Reply:
Might make sense when/if we end up using it in multiple places